Sources/Fuzzilli/CompilerSupport/CompilerSupport.swift

```diff
@@ -0,0 +1,143 @@
+// Functions in support of js -> il compilter development
```

Comment:
This will need the copyright header, you can just copy it from one of the other files.

Reply:
Also is this file basically only used by the ILTool? Could this code be moved into that tool then instead?

Reply:
In moving it over, I had to make a number of previously private thing in Sources/Fuzzilli public. Making the changes seemed to make sense to me, but probably should have further review

Reply:
Right, I see. I think it should be Ok to make some fields that are now "internal" (the default protection level) public. My gut feeling is that these kinds of tasks (e.g. proto conversion via `asProtobuf` and the constructors) should be possible to perform "from the outside", so then those fields should probably just be public anyway.